.travis.yml

```diff
@@ -61,6 +61,16 @@ jobs:
         - npx lerna run build
         - npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
 
+    - name: JavaScript native tests
```

Comment:
Not sure if there's been discussion about the naming but, what do you think about s/native/native mobile/? Same in https://github.com/WordPress/gutenberg/pull/16404/files#diff-354f30a63fb0907d4ad57269548329e3R163 below.

Reply:
I named it as the folder containing the configurations, under `test/native`
Same as `JavaScript unit tests` config are under `test/unit`

But not hard feelings.

Reply:
We happened to chat about this over a voice call, and since we are not having an automated way to ping the mobile team yet on failure, let's add the "mobile" hint in the name so the PR author gets a better idea what's the failure related to.